tests/tests_integration/test_cdf_upload_integration.py

```diff
@@ -309,6 +309,7 @@ def test_file_upload_queue(self):
         )
 
         queue.upload()
+        time.sleep(5)  # Reduce risk of CogniteAPIError, 400, Files not uploaded
```

Comment:
This is a bit aggressive, what we've done before is something along the lines of

```
up to 5 times:
    get file metadata from CDF
    if all files are "uploaded", break
    sleep(1)
```

Reply:
Having big sleeps in tests is not only fragile, but also raises the hurdle for adding new tests, which I really would prefer to avoid.

Reply:
I’ll make a small test util function out of the existing 👍